cdk/lib/support-workers.ts

```diff
@@ -178,6 +169,58 @@ export class SupportWorkers extends GuStack {
         });
     };
 
+    const createTypescriptLambda = (
+      lambdaName: string,
+      additionalPolicies: PolicyStatement[] = []
+    ) => {
+      const lambdaId = `${lambdaName}Lambda`;
+      const lambdaTSFile = lambdaId.charAt(0).toLowerCase() + lambdaId.slice(1);
+      const lambda = new GuLambdaFunction(this, lambdaId, {
+        ...lambdaDefaultConfig,
+        app: "support-workers-typescript",
+        fileName: `support-workers.zip`,
+        runtime: Runtime.NODEJS_20_X,
```

Comment:
Maybe we should use Node 22, which is active LTS currently? (Node 20 is in maintenance mode).

Reply:
Yes I'm going to do that actually, I needed to upgrade CDK first though which I did on Friday.